src/Pixie/QueryBuilder/QueryBuilderHandler.php

```diff
@@ -47,14 +47,15 @@ class QueryBuilderHandler
      *
      * @var array
      */
-    protected $fetchParameters = array(\PDO::FETCH_OBJ);
+    protected $fetchParameters = array(PDO::FETCH_OBJ);
```

Comment:
@TCB13 and this is the same thing, as you are using `use PDO` the trailing `\` is not required